CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+
+All members of our community are expected to follow our [Code of Conduct](https://github.com/process-analytics/.github/blob/main/CODE_OF_CONDUCT.md). Please make sure you are welcoming and friendly in all of our spaces.
+
+## Contributions 
+
+There are many ways to contribute:
+
+- help people with the questions they ask on the [Github Issues](https://github.com/process-analytics/mxgraph-sketch-js/issues)
+- submitting bug reports and feature requests in the [Github Issues](https://github.com/process-analytics/mxgraph-sketch-js/issues/new)
+- writing code which can be incorporated into `mxgraph-sketch` itself
+- improving the documentation
+
+## Code and documentation changes guidelines
+
+For all contributions, please respect the following guidelines:
+
+1. If you've noticed a bug or have a feature request, let us know in the [GitHub Issue tracker](https://github.com/process-analytics/mxgraph-sketch-js/issues/new )! So we can confirm the bug or approve your feature, and provide feedback, before starting to code :slightly_smiling_face:
+
+2. Do the changes in your own [fork](CONTRIBUTING.md#fork--create-a-branch) of the code
+
+3. Do not commit changes to files that are irrelevant to your feature or bugfix (eg: `.gitignore`).
+
+4. Provide tests and documentation whenever possible.
+
+5. Be sure you have followed the [code style](https://github.com/process-analytics/bpmn-visualization-js/blob/master/docs/development/development.md#code-style) for the project.
+
+6. Prior opening a Pull Request, ensure the build is fully working by locally running `npm run all` (build, check and
+test everything)
+
+7. Open a [GitHub Pull Request](./docs/development/pull-request.md#open-a-pull-request) with your patches. (**1** pull request = **1** feature or bug)
+   We will review your contribution and respond as quickly as possible. Keep in mind that this is an open source project, and it may take us some time to get back to you. Your patience is very much appreciated.
+
+8. If this is your 1st Pull Request, sign the [Contributor License Agreement](./docs/development/pull-request.md#sign-the-contributor-license-agreement)
+
+9. Be willing to accept criticism and work on improving your code. 
+
+**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on  GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)    
+
+### Fork & create a branch
+
+[Fork mxgraph-sketch](https://help.github.com/articles/fork-a-repo) and create a branch with a descriptive name. 
+
+A good branch name would be (where issue #25 is the ticket you're working on): **25-annotations_to_tasks**
+
+```sh
+git checkout -b 25-annotations_to_tasks
+```
+
+### IDE configuration
+To know how configurate your development environment, see [IDE configuration](https://github.com/process-analytics/mxgraph-sketch-js/blob/master/docs/development/ide-configuration.md)
```

Comment:
currently we don't have docs for this
```suggestion
```

Reply:
It's a bad renaming. I added the link to bpmn-vizualization-js

Reply:
I suggest to remove it as most of the explanation from bpmn-visualization-js do not apply here

Reply:
Done